tests/spread/rockcraft/extension-django/task.yaml

```diff
@@ -1,14 +1,25 @@
 summary: django extension test
 
 environment:
+  SCENARIO/bare_2204: bare-22.04
+  SCENARIO/bare_2404: bare-24.04
   SCENARIO/base_2204: ubuntu-22.04
   SCENARIO/base_2404: ubuntu-24.04
 
 execute: |
 
   run_rockcraft init --name example-django --profile django-framework
 
-  sed -i "s/base: .*/base: ${SCENARIO//-/@}/g" rockcraft.yaml
+  if [[ "${SCENARIO}" == bare* ]]; then
```

Comment:
here's a suggestion that might clean this test up and make it easier to read:

have both `base` and `build-base` on rockcraft.yaml:
```
base: TEST_BASE
build-base: TEST_BUILD_BASE
```

then set up the `environment` so that you have two variables per test:

```
environment:
  BASE/bare_2204: bare
  BASE/bare_2404: bare
  BASE/base_2204: ubuntu@22.04
  BASE/base_2404: ubuntu@24.04

  BUILD_BASE/bare_2204: ubuntu@22.04
  BUILD_BASE/bare_2404: ubuntu@24.04
  BUILD_BASE/base_2204: ubuntu@22.04
  BUILD_BASE/base_2404: ubuntu@24.04
```

...then you'll still have 4 tests, but each test will have the BASE and BUILD_BASE variables ready to replace.